bin/mongodb/fix-normalized-emails.js

```diff
@@ -1,13 +1,23 @@
-function gmailNormalize(email) {
+const gmailOrProton = ['protonmail.com', 'protonmail.ch', 'pm.me', 'gmail.com', 'googlemail.com'];
+
+function normalize(email) {
   let [name, domain] = email.toLowerCase().split('@');
   [name] = name.split('+');
-  return name.replace(/\./g, '') + '@' + domain;
+
+  if (gmailOrProton.includes(domain)) {
+    return name.replace(/\./g, '') + '@' + domain;
+  } else {
+    return name + '@' + domain;
+  }
 }
 
 db.user4
-  .find({ email: /[^+.]+[+.].*@(protonmail\.com|protonmail\.ch|pm\.me|gmail\.com|googlemail\.com)$/i })
+  .find({
+    email:
+      /([^+.]+[+.].*@(protonmail\.com|protonmail\.ch|pm\.me|gmail\.com|googlemail\.com)|^[^+]+\+.*@.+)$/i,
+  })
```

Comment:
if we're going to remove the `+` name suffixes for all domains, why are we only migrating those?

Reply:
The regex is extended at the end to match all domains if the email has a + in it.

Because I didn't want to undo previous normalizations, I extended the script to add +-normalization on top of the existing behaviour, rather than replacing behaviour. However I think that in practice, for this migration, we could shrink down the regex to only keep the second part, as long as `normalize` still does dot-normalization for gmail/protonmail.